db/fixtures/paths/foundations/courses/beyond_foundations.rb

```diff
@@ -0,0 +1,40 @@
+########################
+# Course - Beyond Foundations
+########################
+course = @path.add_course do |course|
+  course.title = 'Beyond Foundations'
+  course.description = "With the foundational knowledge you've learned, you're almost ready to take on the rest of the curriculum. All that's left is to get a quick introduction to the backend and then a little help in choosing your future with TOP."
+  course.identifier_uuid = '7307ca83-e4cf-4789-b957-d4bdcfe3e200'
+  course.show_on_homepage = true
+  course.badge_uri = ''
```

Comment:
To add the badge, you need to put the name of the file here. These currently live at `app/assets/images` - see there for naming conventions. 

You will also need to add a border-less version at `app/assets/images/badges`. It needs to follow a specific naming convention to work - `app/assets/images/wombat.svg` would need a corresponding `app/assets/images/badges/wombat-borderless.svg`.

Reply:
Ah yeah I noticed the badge icons there, but thank you for the info about the borderless versions! I haven't created badges for the two other courses I created here yet; that was the "not super urgent" bit I mentioned on Discord 😆 I figure I'd throw a draft up first and see what people think before getting badges done. Might depend whether people like this update, and if so whether it'd make sense to implement it now or wait until a potential JS refresh can occur to get them in around the same time.

Reply:
@thatblindgeye No stress at all, I just wanted to get the information out of my head while it was in front of me since how that works is not obvious at all :sweat_smile: 

If you wanted a placeholder, putting `path.badge_uri = 'badge-foundations.svg' here would probably do the trick